<launch>
  <!-- <param name="/use_sim_time" value="true" /> Not sure if this is needed -->
  <node pkg="rosbridge_server" name="rosbridge" type="rosbridge_websocket.py">
    <param name="unregister_timeout" value = "10.0"/>
  </node>
  <node pkg="long_term_deployment" type="task_server.py" name="task_server" output="screen" required="true"/>
  <node pkg="long_term_deployment" type="central_planner.py" name="central_planner" output="screen" required="true"/>

  <node pkg="long_term_deployment" type="map_manager.py" name="map_manager" output="screen"/>
  <!--
  <include file="$(find cardboard_detection_task)/launch/cardboard_query_server_example.launch"/>
  -->
</launch>
